hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -40,8 +40,12 @@
   public static final CacheConfig DISABLED = new CacheConfig();
 
   /**
-   * Configuration key to cache data blocks on read. Bloom blocks and index blocks are always be
-   * cached if the block cache is enabled.
+   * Configuration key to turn on block cache. There are separate switches for read and write.
+   */
+  public static final String BLOCKCACHE_ENABLED = "hbase.block.enabled";
+
```

Comment:
Did you meat some issues that make you separate these configs to resolve them? I don't think an additional  "hbase.block.enabled" is needed, this will make users confused with which meaning of 'block' here. For the meta blocks the cache should not be disabled, and this is the original design.

Reply:
Oh, I got your point. Let me consider this for a while.

Reply:
Right, these changes seems a little bit superfluous. In this case, may I just close this MR and the corresponding jira ticket?

Reply:
OK, thanks.